packages/block-library/src/image/transforms.js

```diff
@@ -141,15 +141,11 @@ const transforms = {
 						( file ) => file.type.indexOf( 'image/' ) !== 0
 					)
 				) {
-					const { createWarningNotice } = dispatch( noticesStore );
-					createWarningNotice(
+					const { createErrorNotice } = dispatch( noticesStore );
+					createErrorNotice(
 						__(
 							'If uploading to a gallery all files need to be image formats'
-						),
-						{
```

Comment:
The unique id stop multiple notices only seems to work for snackbar notices

Reply:
Adding the unique ID stops multiple notices for me within the default editor context. Unfortunately, as the gallery component's notice list is via internal state thanks to the `withNotices` HoC, I couldn't find a means to access it and remove other notices.

Essentially, a user can now get a max of two error notices regarding invalid files being uploaded. One when they drop the invalid file on the placeholder. The second if they drag invalid files into the gallery between other images.